BUILD_MESON

```diff
@@ -28,24 +42,17 @@ in Meson you need to prepare "Buidling Dir". simply just type:
 
     meson setup builddir/
 
-This command configures "Building dir" in builddir/.
+This command configures "Building directory" in builddir/.
```

Comment:
`"Building directory"` should be `the "build directory"`.

Reply:
Please keep line lengths consistent. This line is short, compared to the others.